fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -153,6 +154,8 @@ public HivePartition load(PartitionCacheKey key) throws Exception {
      * generate a filecache and set to fileCacheRef
      */
     public void setNewFileCache() {
+        // init or refresh job conf
+        jobConf = getJobConf();
```

Comment:
I'm not sure if this change will bring race condition or not. Because the reference of jobConf in getFilesByTransaction and loadFiles may be changed at any time. But I don't think it's a big problem since the refresh frequency is very low.

Reply:
add synchronized for getJobConf to avoid this.